src/component/bom.js

```diff
@@ -2548,6 +2620,42 @@ const RescoringCell = ({
               />
             }
           </List>
+          {datasource === datasources.OS_ID && osInfo && <>
+            <List>
+              {emptyOsId ? (
+                <Typography
+                  variant="inherit"
+                  sx={{
+                    whiteSpace: 'pre-wrap',
+                    maxWidth: 'none',
+                  }}
+                >
+                  Unable to determine an OS, thus probably a scratch image.
```

Comment:
just to be sure: this is redundant on purpose, as we will cleanup old entries (and code for those) later - please confirm